src/core/qgsattributeeditorelement.h

```diff
@@ -470,4 +471,48 @@ class CORE_EXPORT QgsAttributeEditorQmlElement : public QgsAttributeEditorElemen
     QString mQmlCode;
 };
 
+
+/**
+ * \ingroup core
+ * An attribute editor widget that will represent arbitrary HTML code.
+ *
+ * \since QGIS 3.10
+ */
+class CORE_EXPORT QgsAttributeEditorHtmlElement : public QgsAttributeEditorElement
+{
+  public:
+
+    /**
+     * Creates a new element which can display HTML
+     *
+     * \param name         The name of the widget
+     * \param parent       The parent (used as container)
+    */
+    QgsAttributeEditorHtmlElement( const QString &name, QgsAttributeEditorElement *parent )
+      : QgsAttributeEditorElement( AeTypeHtmlElement, name, parent )
+    {}
+
+    QgsAttributeEditorElement *clone( QgsAttributeEditorElement *parent ) const override SIP_FACTORY;
+
+    /**
+     * The QML code that will be represented within this widget.
+     *
+     * \since QGIS 3.4
+     */
+    QString htmlCode() const;
+
+    /**
+     * The HTML code that will be represented within this widget.
+     *
+     * @param htmlCode
```

Comment:
```suggestion

```

Either call it `\param` and give it some content, or better drop it (and add a doxygen description inline)

Reply:
:(  I just copied and did a search/replace from https://github.com/qgis/QGIS/blame/b165258e6d852856ab5380b4c0cacf7f89e32480/src/core/qgsattributeeditorelement.h#L464